src/platform/Ameba/NetworkCommissioningWiFiDriver.cpp

```diff
@@ -228,8 +228,7 @@ void AmebaWiFiDriver::OnScanWiFiNetworkDone()
         return;
     }
 
-    rtw_scan_result_t * ScanResult = (rtw_scan_result *) pvPortMalloc(NumAP * sizeof(rtw_scan_result));
-    matter_get_scan_results(ScanResult, NumAP);
+    rtw_scan_result_t * ScanResult = matter_get_scan_results();
```

Comment:
This doesn't seem like "update CI" bits....

Reply:
Yes, this commit is actually to fix the CI error after update to 0.7.0.